web/packages/teleterm/src/services/config/configService.ts

```diff
@@ -85,12 +85,14 @@ const createAppConfigSchema = (platform: Platform) => {
     'keymap.openQuickInput': omitStoredConfigValue(
       z.string().default(defaultKeymap['open-quick-input'])
     ),
-    'fonts.sansSerifFamily': omitStoredConfigValue(
-      z.string().default(defaultFonts['sansSerif'])
-    ),
-    'fonts.monoFamily': omitStoredConfigValue(
-      z.string().default(defaultFonts['mono'])
-    ),
+    /**
+     * This value can be provided by the user and is unsanitized. This means that it cannot be directly interpolated
+     * in a styled component or used in CSS, as it may inject malicious CSS code.
+     * Before using it, sanitize it with `CSS.escape` or pass it as a `style` prop.
+     * Read more https://frontarm.com/james-k-nelson/how-can-i-use-css-in-js-securely/.
+     */
+    'terminal.fontFamily': z.string().default(defaultTerminalFont),
+    'terminal.fontSize': z.number().int().min(5).max(100).default(15),
```

Comment:
Ah damn, forgot to say that I don't think we need to be as strict here, we could do like min 1 and max, idk, 256. I'd rather allow super tiny values since it doesn't change anything for us but might enable some niche use cases for the users.

Reply:
I think we can do it.